lib/rules/aria-allowed-attr.json

```diff
@@ -7,7 +7,7 @@
     "description": "Ensures ARIA attributes are allowed for an element's role",
     "help": "Elements must only use allowed ARIA attributes"
   },
-  "all": [],
-  "any": ["aria-allowed-attr"],
+  "all": ["aria-allowed-attr", "aria-conditional-attr"],
```

Comment:
I really don't like "none" checks.  Always messes with my head that `false` means `pass`. Can't undo that on the other checks but I don't want to make it worse.